contracts/staking/utils/StakingRewardsPerEpoch.sol

```diff
@@ -13,37 +13,34 @@ contract StakingRewardsPerEpoch is ReentrancyGuard, Pausable {
 	struct StakerInfo {
 		uint256 reward; // the reward amount which should be transfered to the user
 		uint256 balance; // the amount of active stake of user
-		uint256 pendingStake; // the amount of pending stake of user
-		uint256 indexOfLastEpochStaked; // an index of the epoch from which the reward is calculated
+		uint256 rewardPerTokenPaid; // the amount of pending stake of user
 	}
 
-	// staker - an address of the staker, amount - an amount of staked tokens, epoch - an epoch when the stake was made
-	event Staked(address indexed staker, uint256 amount, uint256 epoch);
+	// staker - an address of the staker, amount - an amount of staked tokens
+	event Staked(address indexed staker, uint256 amount);
```

Comment:
if we already have events in G$ staking no need for duplicates

Reply:
Removed